apps/dashboard/app/(app)/identities/layout.tsx

```diff
@@ -21,7 +22,7 @@ export default async function AuthorizationLayout({
     return redirect("/auth/sign-in");
   }
 
-  if (!workspace.betaFeatures.identities) {
+  if (getFlag(workspace, "identities", { prodFallback: true, devFallback: false })) {
```

Comment:
you flipped it around, so when they had it set to true in our database, we would actually deny them access
```suggestion
  if (!getFlag(workspace, "identities", { prodFallback: false, devFallback: true })) {
```